src/app/orders/(components)/CommentBox.tsx

```diff
@@ -0,0 +1,91 @@
+import { useState } from 'react';
+import styles from './CommentBox.module.scss';
+
+interface Comment {
+    id: number;
+    name: string;
+    text: string;
+    timestamp: string;
+}
+
+const CommentBox = () => {
+    const [comment, setComment] = useState<string>('');
+    const [comments, setComments] = useState<Comment[]>([]);
+    const [editingCommentId, setEditingCommentId] = useState<number | null>(null);
+
+    // Static commenter names for demonstration
+    const commenterNames = ["Rubin", "Himashu", "Yugant", "Kartik","Yuvraj"];
+
+    const handleCommentChange = (event: React.ChangeEvent<HTMLTextAreaElement>) => {
+        setComment(event.target.value);
+    };
+
+    const handleSubmit = () => {
+        if (comment.trim()) {
+            const name = commenterNames[Math.floor(Math.random() * commenterNames.length)];
+
+            if (editingCommentId !== null) {
+                setComments(comments.map(c =>
+                    c.id === editingCommentId ? { ...c, text: comment, timestamp: new Date().toLocaleString() } : c
+                ));
+                setEditingCommentId(null);
+            } else {
+                const newComment = {
+                    id: Date.now(),
+                    name: name,
+                    text: comment,
+                    timestamp: new Date().toLocaleString(),
+                };
+                setComments([newComment, ...comments]);
+            }
+            setComment('');
+        }
+    };
+
+    const handleEdit = (id: number) => {
+        const commentToEdit = comments.find(c => c.id === id);
+        if (commentToEdit) {
+            setComment(commentToEdit.text);
+            setEditingCommentId(id);
+        }
+    };
+
+    const handleDelete = (id: number) => {
+        setComments(comments.filter(c => c.id !== id));
+    };
+
+    return (
+        <div className={styles.commentBox}>
+      <textarea
+          value={comment}
+          onChange={handleCommentChange}
+          rows={4}
+          className={styles.textarea}
+          placeholder="Write your comment here..."
+      />
+            <button onClick={handleSubmit} className={styles.button}>
+                {editingCommentId !== null ? 'Update' : 'Submit'}
+            </button>
+
+            <div className={styles.commentsList}>
+                {comments.map(comment => (
+                    <div key={comment.id} className={styles.comment}>
+                        <strong>{comment.name}</strong> {/* Display the commenter's name */}
+                        <p className={styles.commentText}>{comment.text}</p>
```

Comment:
```suggestion
                        <p className={styles.commentText text-xl p-2 m-2}>{comment.text}</p>
```
Use inline tailwind attributes for font-size, padding, margin